### PR TITLE
Feat/local storage update

### PR DESCRIPTION
**Pull request by raz-joan**
What (if any) features are you implementing?
- local storage of our global state object
- 
What (if anything) did you refactor?
- FavoriteDrinks reducer
- add keys to <p> in the Drink component ingredient list
- Add clearInputs to Return to Main button on PairedPage
- 
Were there any issues that arose?
-
Is there anything that you need from your teammate?
-
Any other comments, questions, or concerns?
- Thanks for navigating these fixes together!

### CODE REVIEW

File: src/Components/Drink.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const Drink = () => {
 
     const ingredientsList = recipe.ingredients.map((ingredient, index) => {
         return (
-            <p>{`${recipe.measures[index]} ${ingredient}`}</p>
+            <p key={index}>{`${recipe.measures[index]} ${ingredient}`}</p>
```
Comment (colganmeanor) [inline src/Components/Drink.js:54]: this was a very good solution to our unique key issue. Thank you Joan!

File: src/redux/reducers/favoriteDrinkReducer.js [hunk 2]
```diff
@@ -6,9 +6,14 @@ const initialState = {
 export const favoriteDrinksReducer = (state = initialState, action) => {
     switch (action.type) {
         case 'ADD_FAVORITE_DRINK':
-            return {
-                ...state,
-                favoriteDrinks: [...state.favoriteDrinks, action.payload]
+            const foundDrink = state.favoriteDrinks.find(drink => drink.idDrink === action.payload.idDrink)
```
Comment (colganmeanor) [inline src/redux/reducers/favoriteDrinkReducer.js:9]: glad that this worked! bye bye duplicate favorites!

**Review by colganmeanor — APPROVED** [review]

File: src/Components/Drink.js (hunk 1 above)
Comment (michelecomfort) [inline src/Components/Drink.js:54]: Nice problem solving!

File: src/Components/PairedPage.js [hunk 3]
```diff
@@ -19,6 +20,11 @@ const PairedPage = () => {
     const game = useSelector(state => state.boardGame.currentGame)
     const currentDrink = useSelector(state => state.favoriteDrinks.currentDrink)
 
+    const clearInputs = () => {
```
Comment (michelecomfort) [inline src/Components/PairedPage.js:23]: Good catch!

File: src/redux/reducers/favoriteDrinkReducer.js (hunk 2 above)
Comment (michelecomfort) [reply src/redux/reducers/favoriteDrinkReducer.js:9]: ^^ what Colgan said.

File: src/redux/store.js [hunk 4]
```diff
@@ -9,7 +9,12 @@ const rootReducers = {
     boardGame: boardGameReducer
 }
 
-const store = createStore(combineReducers(rootReducers), window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
+const storedState = localStorage.getItem('globalState') ? JSON.parse(localStorage.getItem('globalState')) : {}
 
-export default store
+const store = createStore(combineReducers(rootReducers), storedState)
+
+store.subscribe(() => {
```
Comment (michelecomfort) [inline src/redux/store.js:16]: Love the simplicity of using local storage within redux.

**Review by michelecomfort — COMMENTED** [review]
Almost there folks!

**Pull request merged**